test/parallel/test-repl-sigint.js

```diff
@@ -45,6 +37,12 @@ child.stdout.once('data', common.mustCall(() => {
 
 child.on('close', function(code) {
   assert.strictEqual(code, 0);
-  assert.notStrictEqual(stdout.indexOf('Script execution interrupted.\n'), -1);
-  assert.notStrictEqual(stdout.indexOf('42042\n'), -1);
+  assert.ok(
+    stdout.includes('Script execution interrupted.\n'),
+    `Expected stdout to contain "Script execution interrupted.", got ${stdout}`
```

Comment:
I left it out because a literal `\n` in the message output would probably be unhelpful and an escaped `\n` might be taken to mean that an escaped `\n` is what is expected.

I'm OK with any of the three possibilities (leave it out, put it in literally, put it in escaped). I chose this one (leave it out) for simplicity. If you feel that it should be there, let me know if you want it escaped or not. And if, like me, you don't feel particularly strongly, then I'll probably just leave it as it is. ¯\\_(ツ)_/¯